vendor/bat-native-ledger/include/bat/ledger/publisher_info.h

```diff
@@ -137,6 +137,17 @@ LEDGER_EXPORT struct PublisherInfo {
   std::vector<ContributionInfo> contributions;
 };
 
+LEDGER_EXPORT struct PublisherInfoListStruct {
+  PublisherInfoListStruct();
+  ~PublisherInfoListStruct();
+  PublisherInfoListStruct(const PublisherInfoListStruct& data);
+
+  const std::string ToJson() const;
+  bool loadFromJson(const std::string& json);
+
+  std::vector<PublisherInfo> list_;
```

Comment:
struct members should not have trailing underscores

Reply:
but why do we need a struct to hold a PublisherInfoList?

Reply:
because it needs `ToJson` and `loadFromJson` for mojo conversion

Reply:
and I can't append this two functions to `use` of `type`

Reply:
changed `list_` into `list`